egs/heroico/s5/local/score_combine.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
this could be probably deleted too, you don't seem to use the lattice-combination anywhere